js-createelement_toast-messages/js/index.js

```diff
@@ -4,10 +4,17 @@ const toastContainer = document.querySelector('[data-js="toast-container"]');
 const addButton = document.querySelector('[data-js="add-button"]');
 const clearButton = document.querySelector('[data-js="clear-button"]');
 
-addButton.addEventListener("click", () => {
+addButton.addEventListener("click", (event) => {
+  event.preventDefault();
```

Comment:
`event.preventDefault()` is not needed here I think. If this was a form and we used submit event, we would add preventDefault for the form not to do GET request and stop the reload.

Otherwise looks good! 👏